packages/autocomplete-preset-algolia/src/search/getAlgoliaInsightsHits.ts

```diff
@@ -0,0 +1,32 @@
+import { Hit } from '@algolia/client-search';
+
+import { search, SearchParams } from './search';
+
+export function getAlgoliaInsightsHits<TRecord>({
+  searchClient,
+  queries,
+}: SearchParams): Promise<Array<Array<Hit<TRecord>>>> {
+  const insightsQueries = queries.map((query) => ({
+    ...query,
+    params: {
+      ...query.params,
+      clickAnalytics: true,
+    },
+  }));
+
+  return search<TRecord>({ searchClient, queries: insightsQueries }).then(
+    (response) => {
+      const results = response.results;
+
+      return results.map((result) =>
+        result.hits.map((hit) => {
+          return {
+            ...hit,
+            __autocomplete_indexName: result.index,
```

Comment:
if we did this for the regular hits, the only change people would have to make is adding clickAnalytics: true to the params. I wonder if it's simpler to keep the regular getAlgoliaHits

Reply:
Good question.

Either way is fine to me. Gonna wait for @eunjae-lee POV on that.

Reply:
I also think it's good to have only `getAlgoliaHits`. Otherwise, we're gonna want to have `getAlgoliaPersonalizedHits` in the future.

Reply:
Addressed in 11fcb2c2ae4431e688473232662742146cd78de5.